wikimedia/wikipedia-ios
# AaaLD - Navigate to particular talk page topic if possible

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T267849

### Notes
* This is using the talk page topic section title in the significant events models to navigate the user to a specific topic after tapping "View discussion" by passing it in as a fragment. If there is an issue with generating the fragment, default to the article talk page like it was before. 

### Test Steps
1. Go to AaaLD experiment for various articles and tap "View discussion".
2. Confirm it goes directly to the article talk page topic rather than the homepage.
3. If it doesn't, check to see if that topic still exists on desktop - it may have been archived in which case there's not much we can do. If it does exist, please mention as a bug comment on this PR. It's likely something with the encoding (which has been finicky).
